api/test/robot_tests/cmd/keywords.resource

```diff
@@ -26,7 +26,7 @@ Delete Wallet Plugin
 Install Wallet Plugin
     Log To Console    Installing Massa Wallet Plugin
     ${source}=    Set Variable
-    ...    https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.12/wallet-plugin_${OS}-${ARCH}.zip
+    ...    https://github.com/massalabs/thyra-plugin-wallet/releases/download/v0.0.13/wallet-plugin_${OS}-${ARCH}.zip
```

Comment:
Can't you use a constant and set it at the begining of the file instead ?

Reply:
I put it in this file api/test/robot_tests/plugin_manager/variables.resource